things/service.go

```diff
@@ -283,13 +283,13 @@ func (ts *thingsService) RemoveChannel(ctx context.Context, token, id string) er
 	return ts.channels.Remove(ctx, res.GetValue(), id)
 }
 
-func (ts *thingsService) Connect(ctx context.Context, token, chID string, thIDs ...string) error {
+func (ts *thingsService) Connect(ctx context.Context, token string, chIDs []string, thIDs []string) error {
```

Comment:
This could be `owner string, chIDs, thIDs []string`

Reply:
There's a few of these.  I'll switch them over.